test/draw/RasterYuv420SemiplanarTest.ooc

```diff
@@ -16,6 +16,26 @@ RasterYuv420SemiplanarTest: class extends Fixture {
 	_inputOddHeight := "test/draw/input/Barn.png"
 	init: func {
 		super("RasterYuv420Semiplanar")
+		this add("yuv fill", func {
+			yuvImage := RasterYuv420Semiplanar new(IntVector2D new(2, 2))
+			yuvImage fill(ColorRgba new(0, 100, 200, 255))
+			yuvSample := yuvImage[0, 0]
+			expect(yuvSample distance(ColorYuv new(81, 194, 69)) < 8.0f)
+			rgbImage := RasterRgb convertFrom(yuvImage)
+			rgbSample := rgbImage[0, 0]
+			expect(rgbSample distance(ColorRgb new(0, 100, 200)) < 8.0f)
```

Comment:
Use `, is less than(` rather than `<` for expect calls in tests. Here and on line 23.